org.eclipse.xtext.tests/src/org/eclipse/xtext/generator/TypeReferenceTest.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2015, 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.generator;
+
+import org.eclipse.emf.ecore.EcorePackage;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.xtext.xtext.generator.model.TypeReference;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
+
+public class TypeReferenceTest {
+	/**
+	 * See https://bugs.eclipse.org/bugs/show_bug.cgi?id=480196
```

Comment:
This is fixed by now though

Reply:
did not want to look into the details thus is just took over the comment